README.md

```diff
@@ -744,6 +816,95 @@ The nearest grocery of SFU is Nesters Market. It is 1.234 miles far, and It is e
+| 9    | 5181876 | Senior Data Analyst-Hardware & Supply Analytics   | Lime              | [{'name': 'Flexible / Remote'}]                   | Senior Level | 2021-02-03T23:02:36.355894Z |
+
+</details>
+
+<details>
+
+  <summary>What are the first 10 senior level data science positions in Amazon?</summary>
+
+  ```python
+from dataprep.connector import connect
+
+# You can get ”app_key“ by following https://www.themuse.com/developers/api/v2/apps
+dc = connect('themuse', _auth={'access_token': app_key})
+
+df = await dc.query('jobs', page=1, category='Data Science', company='Amazon', level='Senior Level')
+df[:10][['id', 'name', 'company', 'locations', 'levels', 'publication_date']]
+  ```
+
+|      | id      | name                                              | company | locations                            | levels       | publication_date            |
+| ---- | ------- | ------------------------------------------------- | ------- | ------------------------------------ | ------------ | --------------------------- |
+| 0    | 5153796 | Sr. Data Architect, Data Lake & Analytics - Na... | Amazon  | [{'name': 'San Diego, CA'}]          | Senior Level | 2021-02-01T22:54:14.002653Z |
+| 1    | 4083477 | Principal Data Architect, Data Lake & Analytics   | Amazon  | [{'name': 'Chicago, IL'}]            | Senior Level | 2021-02-01T23:14:17.251814Z |
+| 2    | 4149878 | Principal Data Architect, Data Warehousing & MPP  | Amazon  | [{'name': 'Arlington, VA'}]          | Senior Level | 2021-02-01T23:15:22.017573Z |
+| 3    | 4497753 | Data Architect - Data Lake & Analytics - Natio... | Amazon  | [{'name': 'Irvine, CA'}]             | Senior Level | 2021-02-01T23:15:22.439949Z |
+| 4    | 4870271 | Data Scientist                                    | Amazon  | [{'name': 'Seattle, WA'}]            | Senior Level | 2021-02-01T23:04:25.967878Z |
+| 5    | 4603482 | Data Scientist - Prime Gaming                     | Amazon  | [{'name': 'Seattle, WA'}]            | Senior Level | 2021-02-01T23:10:37.628292Z |
+| 6    | 5193240 | Data Scientist                                    | Amazon  | [{'name': 'Seattle, WA'}]            | Senior Level | 2021-02-04T23:56:19.176327Z |
+| 7    | 4678426 | Sr Data Architect - Streaming                     | Amazon  | [{'name': 'Roseville, CA'}]          | Senior Level | 2021-02-01T22:51:25.598645Z |
+| 8    | 4150011 | Data Architect - Data Lake & Analytics - Natio... | Amazon  | [{'name': 'Tampa, FL'}]              | Senior Level | 2021-02-04T23:56:18.281215Z |
+| 9    | 4346719 | Sr. Data Scientist - ML Labs                      | Amazon  | [{'name': 'London, United Kingdom'}] | Senior Level | 2021-02-01T23:12:42.038111Z |
+
+</details>
+
+<details>
+  <summary>What are the top 10 companies in engneering?(intelligently sorted by a number of factors such as trendiness, uniqueness, newness, etc)</summary>
```

Comment:
What are the top 10 companies in engneering?(intelligently sorted by a number of factors such as trendiness, uniqueness, newness, etc) -> What are the top 10 companies in engineering? (sorted by factors such as trendiness, uniqueness, newness, etc)